[2020-11-12 12:12:34,386] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: core_concepts.python_function 2020-11-11T00:00:00+00:00 [queued]>
[2020-11-12 12:12:34,396] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: core_concepts.python_function 2020-11-11T00:00:00+00:00 [queued]>
[2020-11-12 12:12:34,396] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-11-12 12:12:34,396] {taskinstance.py:880} INFO - Starting attempt 1 of 1
[2020-11-12 12:12:34,396] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-11-12 12:12:34,402] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): python_function> on 2020-11-11T00:00:00+00:00
[2020-11-12 12:12:34,405] {standard_task_runner.py:53} INFO - Started process 99034 to run task
[2020-11-12 12:12:34,461] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: core_concepts.python_function 2020-11-11T00:00:00+00:00 [running]> 19.1.168.192.in-addr.arpa
[2020-11-12 12:12:34,481] {logging_mixin.py:112} INFO - 0.13436424411240122
[2020-11-12 12:12:34,482] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-11-12 12:12:34,486] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=core_concepts, task_id=python_function, execution_date=20201111T000000, start_date=20201112T171234, end_date=20201112T171234
[2020-11-12 12:12:44,376] {logging_mixin.py:112} INFO - [2020-11-12 12:12:44,375] {local_task_job.py:103} INFO - Task exited with return code 0
